docs/src/doc/user-guide/exporting.md

```diff
@@ -0,0 +1,25 @@
+## Exporting games
+
+To export game, you should copy exports in right template folder, which should be:  
+- Windows: `%APPDATA%\Godot\templates\<godot-version>`
+- Linux: `~/.local/share/godot/templates/<godot-version>`  
+
+You should also renamed the `release` export to:  
+- Windows: `windows_64_release.exe`
+- Linux: `linux_x11_64_release`
+
+And `release_debug` to:
+- Windows: `windows_64_debug.exe`
+- Linux: `linux_x11_64_debug`
+
+On osx use the script mentionned in [this issue](https://github.com/godotengine/godot-docs/issues/3194#issuecomment-588862977),
+this will do the job for you.
+
```

Comment:
Is this also needed when we ship the engine?

Reply:
yep, that would be cool to rename them during release CI

Reply:
I would add a new issue to do this as it's a quality of life improvement